cl/search/models.py

```diff
@@ -2013,13 +2021,31 @@ class Court(models.Model):
         TERRITORY_TRIAL,
         TERRITORY_SPECIAL,
     ]
+    MILITARY_JURISDICTIONS = [
+        MILITARY_APPELLATE,
+        MILITARY_TRIAL,
+    ]
 
     id = models.CharField(
         help_text="a unique ID for each court as used in URLs",
         max_length=15,  # Changes here will require updates in urls.py
         primary_key=True,
     )
-
+    parent_court = models.ForeignKey(
+        "self",
+        on_delete=models.SET_NULL,
+        blank=True,
+        null=True,
+        related_name="subcourts",
+        help_text="Parent court for subdivisions",
+    )
+    appellate_courts = models.ManyToManyField(
+        "self",
+        blank=True,
+        symmetrical=False,
+        related_name="appellate_courts_from",
+        help_text="Appellate courts for this court",
+    )
```

Comment:
Would this be more clear if the name were `appeals_to` and then related could be `appeals_from`?

Reply:
yes.

Reply:
Looks like you didn't include this change.

Reply:
There must be a ghost in git.  I made those changes but will update again

Reply:
ah - I thought you were referring to the related name which I changed.